### DISCUSSION TITLE
How can I write an equation in pool(with())?

### DISCUSSION THREAD

**Discussion by Michael-Covell** [amices/mice#351] [Q&A]
I understand there is a function called `pool.scalar()`, but I'm still getting an error message.  I want to perform an equation with each imputed dataset.  Here is the equation...

```
subset <- subset(df,df$BCC=='True',select=broilers_boosted)
a <- sum(subset,na.rm=TRUE)
b <- sum(df$broilers_boosted,na.rm = TRUE)
a/b
```

Is it possible to do this?

**Comment by stefvanbuuren** [comment]
Sorry, I have no idea what you're trying to do. 

When opening an issue, please provide a reproducible example of a problem or bug in `mice`.
